Lib/test/test_peg_generator/test_pegen.py

```diff
@@ -493,6 +493,14 @@ def test_start_leader(self) -> None:
         # Would assert False without a special case in compute_left_recursives().
         make_parser(grammar)
 
+    def test_opt_sequence(self) -> None:
+        grammar = """
+        start: [NAME*]
+        """
+        # This case was failing because of double trailing comma at the end
+        # of the generated parser. See bpo-41044
```

Comment:
```suggestion
        # This case was failing because of a double trailing comma at the end
        # of a line in the generated source. See bpo-41044
```